### PR TITLE
fix(core): Changes to workflow staticData erroneously updating updatedAt

### PR DESCRIPTION
**Pull request by valya**
## Summary

When workflow `staticData` was changed (usually by a trigger node), it'd cause the `updatedAt` column to update, even though the workflow hadn't really updated.

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/cli/test/integration/services/workflowStaticData.service.test.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+		const workflowEntityNew = await workflowRepository.get({
+			id: workflowEntityOriginal.id,
+		});
+
+		expect(workflowEntityNew?.staticData).toEqual(workflow.staticData);
+		expect(workflowEntityNew?.updatedAt).toEqual(workflowEntityOriginal.updatedAt);
+	});
+
+	it('should not changed workflow updatedAt when calling saveStaticData', async () => {
```
Comment (krynble) [inline packages/cli/test/integration/services/workflowStaticData.service.test.ts:53]:
```suggestion
	it('should not change workflow updatedAt when calling saveStaticDataById', async () => {
```

File: packages/cli/test/integration/services/workflowStaticData.service.test.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+afterAll(async () => {
+	await testDb.terminate();
+});
+
+afterEach(async () => {
+	await testDb.truncate(['User']);
+});
+
+describe('WorkflowStaticDataService', () => {
+	it('should not changed workflow updatedAt when calling saveStaticData', async () => {
```
Comment (krynble) [inline packages/cli/test/integration/services/workflowStaticData.service.test.ts:30]:
```suggestion
	it('should not change workflow updatedAt when calling saveStaticData', async () => {
```

**Review by krynble — DISMISSED** [review]
Minor text changes

**Review by krynble — APPROVED** [review]

**Review by krynble — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.47.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.0)

**Comment by janober** [comment]
Got released with [`n8n@1.47.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.0)

**Pull request merged**